hack/update-generated-completions.sh

```diff
@@ -24,4 +24,7 @@ if [[ ! "$genbashcomp" ]]; then
   exit 1
 fi
 
-os::build::gen-docs "${genbashcomp}" "${OS_ROOT}/rel-eng/completions/bash/"
+
+OUTPUT_DIR_ROOT=${1:-"${OS_ROOT}/rel-eng/completions/"}
+
+os::build::gen-docs "${genbashcomp}" "${OUTPUT_DIR_ROOT}/bash"
```

Comment:
In case there are other completions. If there never will be, we should get rid of the `bash` subdir.